tests/DocumentationExamplesTest.php

```diff
@@ -1408,6 +1410,57 @@ public function testTransactions_retry_example_3()
         }
     }
 
+    function testCausalConsistency()
+    {
+        $this->skipIfCausalConsistencyIsNotSupported();
+
+        // Prep
+        $client = new Client($this->getUri());
+        $test = $client->selectDatabase(
+            'test',
+            [ 'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY) ]
```

Comment:
This is not within the example code, so we can use `WriteConcern` here (it's already `use`-ed up top).